src/app/component/SliderWidget/SliderWidget.component.js

```diff
@@ -150,14 +139,18 @@ export class SliderWidget extends PureComponent {
         const { activeImage } = this.state;
         const { slider: { slides, title: block } } = this.props;
 
+        const children = slides.map(this.renderSlide);
+        const childrenClones = slides.map((el, i) => this.renderSlide(el, i + children.length));
```

Comment:
As I mentioned before, can this logic be transported to the Slider component itself?

Reply:
And I believe React explicitly warns every single time one tries to use an index as a key. Indexes are NOT valid for use as keys. Please, read React docs and explain why to me personally in Slack!